src/MyEventEmitter.js

```diff
@@ -1,12 +1,93 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  addEvent(name, callback, repeat, prepend) {
+    if (!Object.hasOwnProperty.call(this.events, name)) {
+      this.events[name] = [
+        {
+          repeat,
+          callback,
+        },
+      ];
+
+      return;
+    }
+
+    if (prepend) {
+      this.events[name].unshift({
+        repeat,
+        callback,
+      });
+    } else {
+      this.events[name].push({
+        repeat,
+        callback,
+      });
+    }
+  }
+
+  on(name, callback) {
+    this.addEvent(name, callback, true, false);
```

Comment:
When you have 3 and more arguments it's better to pass them using an object. Currently, it's hard to read, especially two booleans in the end...